.gitignore

```diff
@@ -21,3 +21,9 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+####
+.lintrc
+.stylelintrc.json
+.hintrc
+.eslintrc.json
```

Comment:
- [ ] Even though you have put these files in the .gitignore I still see them in the Github repository, please remove them. 😸

Reply:
Alright